pkg/bazel/bazel.go

```diff
@@ -94,3 +95,31 @@ func (b *bazel) Flags() (map[string]*FlagInfo, error) {
 
 	return flags, nil
 }
+
+func (b *bazel) AQuery(query string) (*ActionGraphContainer, error) {
```

Comment:
Missing godoc. Actually, we need to add more docs to the public API of this package.